src/db/tripledb.plt

```diff
@@ -1,54 +1,69 @@
 :- begin_tests('tripledb').
-
 :- use_module(library('db/tripledb'),
     [ tripledb_load/2,
       tripledb_ask/3,
       tripledb_forget/3,
       tripledb_tell/3
     ]).
 
+:- use_module(library('semweb/rdf_db'),  [ rdf_register_ns/3 ]).
+
+% register namespace for following tests
+:- rdf_register_ns(swrl_tests,'http://knowrob.org/kb/swrl_test#',[keep(true)]).
+
 % load swrl owl file for tripledb testing
 test('tripledb_load_local_owl_file(URL)') :-
-  tripledb_load(
-    'package://knowrob/owl/test/swrl.owl',
-    [ graph(common),
-      namespace(knowrob, 'http://knowrob.org/kb/swrl_test#')
-    ]).
+    tripledb_load(
+        'package://knowrob/owl/test/swrl.owl',
```

Comment:
loading the swrl ontology into common graph without removing it after the test might create problems in other tests. Other tests would load the OWL file into the graph named "test" -- probably best to use it here as well.

Reply:
test graph is used to load triples from swrl.owl